lib/initialize.js

```diff
@@ -75,10 +75,10 @@ module.exports = function initialize(hook, sails, done){
 
         });//</_.each>
 
-        return next(undefined, relevantDatastoreNames);
 
       } catch (e) { return next(e); }
 
+      return next(undefined, relevantDatastoreNames);
```

Comment:
Note for reference: This presumably fixes the bad error message like this one: 

![image](https://cloud.githubusercontent.com/assets/618009/22994520/d5acc068-f38c-11e6-861c-a648750ad56b.png)

^^that was smothering the _real_ error msg